openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,53 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from model_utils.models import TimeStampedModel
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(TimeStampedModel):
+    """
+    Model to save invitations, sent out to users for becoming admin of business unit(s).
```

Comment:
*Model to save invitations sent out to users to grant them ADG admin or business unit admin roles.